bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -359,6 +359,17 @@ impl Client {
 
 #[matrix_sdk_ffi_macros::export]
 impl Client {
+    /// Perform database optimizations if any are available, i.e. vacuuming in
+    /// SQLite.
+    pub async fn optimize_stores(&self) -> Result<(), ClientError> {
+        Ok(self.inner.optimize_stores().await?)
+    }
+
+    /// Returns the sizes of the existing stores, if known.
+    pub async fn get_store_sizes(&self) -> Result<StoreSizes, ClientError> {
```

Comment:
We usually don't have a `get_` prefix, only a `set_` prefix. It's not the end of the world, but so you know, it's more Rust idiomatic :-).